cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -139,7 +140,7 @@ func (cc *createClusterOptions) createCluster(cmd *cobra.Command, _ []string) er
 		WithBootstrapper().
 		WithClusterManager(clusterSpec.Cluster).
 		WithProvider(cc.fileName, clusterSpec.Cluster, cc.skipIpCheck, cc.hardwareFileName, cc.skipPowerActions, cc.setupTinkerbell, cc.forceClean).
-		WithFluxAddonClient(clusterSpec.Cluster, clusterSpec.FluxConfig).
+		WithFluxAddonClient(clusterSpec.Cluster, clusterSpec.FluxConfig, cliConfig).
```

Comment:
I could see the cliConfig change in the future where if we need to pass in env vars from the controller and build this client, we would have a different abstraction here. But since the only uses of this method is from the CLI, I wouldn't worry about this now.